datasm/tools/run_cwltool_test.sh

```diff
@@ -6,12 +6,13 @@ in_cmip_dsid=$1
 dry_run=1
 
 # external dependencie
-get_cmip6_parms=/p/user_pub/e3sm/staging/tools/cmip6_parameters.sh
-force_status=/p/user_pub/e3sm/staging/tools/ensure_status_file_for_dsid.sh
-list_e3sm_dsids=/p/user_pub/e3sm/staging/tools/list_e3sm_dsids.py
-meta_version=/p/user_pub/e3sm/staging/tools/metadata_version.py
+dsm_tools=`$DSM_GETPATH STAGING_TOOLS`
+resource_path=`$DSM_GETPATH STAGING_RESOURCE`
```

Comment:
I noticed similar paths get repeated in other tools. 

Is it possible to consolidate these paths into a single file then reference them based on the needs of each file?
This will cute down on repeated code and will probably make maintaining the tools easier.